packages/service-integration/src/service_integration/osparc_config.py

```diff
@@ -148,6 +154,14 @@ class SettingsItem(BaseModel):
         description="The value of the service setting (shall follow Docker REST API scheme for services",
```

Comment:
Reading this now, I am not sure I understand why it's rewritten.
Also, the new concept is to run containers, not services (at least from the sidecars) so the description is actually incorrect. sorry for that. see https://docs.docker.com/engine/api/v1.41/#operation/ContainerCreate

Reply:
LEt's double check that tomorrow

Reply:
Please remember this is also used for validating `dynamic-services` as well.